qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/QuteDefinition.java

```diff
@@ -270,6 +273,72 @@ private CompletableFuture<List<? extends LocationLink>> findDefinitionFromPart(P
 		}
 	}
 
+	/**
+	 * Find parameter definition for Enum in #when section.
+	 *
+	 * @param parameter
+	 * @param template
+	 * @param cancelChecker
+	 *
+	 * @return parameter definition for Enum in #when section.
+	 */
+	private CompletableFuture<List<? extends LocationLink>> findDefinitionFromParameter(Parameter parameter,
+			Template template, CancelChecker cancelChecker) {
+		if (!isCaseSection(parameter.getParentSection())
+				|| BaseWhenSection.isOperator(parameter.getName()) && isFirstParameter(parameter)) {

```

Comment:
Déclare 

Section caseSection = parameter.getOwnerSection();

And works with variable in the following code